04 Time to Practice - Component Basics/01-starting-setup/src/App.js

```diff
@@ -0,0 +1,19 @@
+// Creating a Demo App for diplaying data 
+
+// Practiced topics : Component Creation, Component Splitting and Data Display. 
+
+import React from 'react';
+import Header from './components/Header/Header';
+import Concepts from './components/Concepts/Concepts';
+import * as utils from './utils';
+
+function App() {
+  return (
+    <div>
+      <Header/>
+      <Concepts item={utils.concepts}/>
```

Comment:
should always use plural (items) in naming property if it will store more than 1 property.